cli/src/cli-rpc-ops.c

```diff
@@ -10478,8 +10478,8 @@ gf_cli_get_vol_opt_cbk(struct rpc_req *req, struct iovec *iov, int count,
         goto out;
     }
 
-    cli_out("%-40s%-40s", "Option", "Value");
-    cli_out("%-40s%-40s", "------", "-----");
+    cli_out("%-50s%-40s", "Option", "Value");
```

Comment:
And what if we have an option of 50 or more characters ?

There's no value that guarantees that all texts will fit in a column, so either we compute the maximum width and format the table accordingly in a dynamic way, or we assume that some columns will overflow the reserved space. That's not a big problem as long as we put a separator between both columns (currently there's no separator, so if there's overflow both columns get concatenated. We should put an space at least).

Reply:
If that's the case, even a separator won't guarantee a properly aligned table.
In such a case, I think we might have to take out the max length of the key each time and then use that value to print the table. Should that be done?

Or can we ignore the case of any option having a length of more than 50 characters as I consider that probability of having such an option is very less because even now we have only 1-3 options at max which are more than 40 characters?

Reply:
IMO it's not critical to have the table aligned. As long as there's a separator, it can be easily parsed by scripts. We cannot guarantee that any future option won't have more than 49 characters.

Also note that the original output was formatted to use the standard 80 columns width. I'm not sure if it's important to keep it or not.

Reply:
> IMO it's not critical to have the table aligned. As long as there's a separator, it can be easily parsed by scripts. We cannot guarantee that any future option won't have more than 49 characters.

Ok, then a separator like `-` or `:` should do?

> Also note that the original output was formatted to use the standard 80 columns width. I'm not sure if it's important to keep it or not.

I checked the code for any such reference, but couldn't find any.

Reply:
I have updated the code to use `:` as a separator and keep the 80 columns width, be default.

Reply:
> > IMO it's not critical to have the table aligned. As long as there's a separator, it can be easily parsed by scripts. We cannot guarantee that any future option won't have more than 49 characters.
> 
> Ok, then a separator like `-` or `:` should do?

I would use a simple space. We are separating columns. Adding special characters may be confused with the contents of the columns themselves.

Considering that first column is an option name and it won't contain any space, the space should be good enough.

> 
> > Also note that the original output was formatted to use the standard 80 columns width. I'm not sure if it's important to keep it or not.
> 
> I checked the code for any such reference, but couldn't find any.

80 columns width is some sort of standard beyond Gluster. Sticking to it when possible could be helpful for some users that could still depend on it. However I don't have a strong feelings about it if others are ok to increase the width.